kernel/Dockerfile

```
@@ -26,8 +25,7 @@ RUN apk add \
     tar \
     xz \
     xz-dev \
-    zlib-dev \
-    || true
```

but it also masks errors